scenegraph/Scene.cpp

```diff
@@ -81,6 +85,7 @@ void Scene::parseNode(Scene *sceneToFill, CS123SceneNode *node, glm::mat4x4 comp
         sceneToFill->parseNode(sceneToFill, c[i], composite);
     }
 
+    sceneToFill->loadTextures();
```

Comment:
I think this could be faster if it's added in Scene::parse, instead of parseNode?